Library/ViewModels/PostCampaignPledgeRewardsSummaryViewModel.swift

```diff
@@ -211,7 +211,7 @@ private func items(
 
   // MARK: Bonus
 
-  if let bonus = bonusAmount, bonus > 0 {
+  if let bonus = bonusAmount, bonus > 0, rewardItems.isEmpty == false {
```

Comment:
Sanity checking this shouldn't be doing that same `isNoReward` check here

Reply:
either or would work here actually.